server/src/main/java/com/linecorp/centraldogma/server/CentralDogmaConfig.java

```diff
@@ -127,14 +145,20 @@ public static String convertValue(@Nullable String value, String propertyName) {
         }
 
         final String prefix = value.substring(0, index);
+        if (!PREFIX_PATTERN.matcher(prefix).matches()) {
+            // Not a prefix.
+            return value;
+        }
+
         final String rest = value.substring(index + 1);
 
         final ConfigValueConverter converter = CONFIG_VALUE_CONVERTERS.get(prefix);
         if (converter != null) {
             return converter.convert(prefix, rest);
         }
-
-        throw new IllegalArgumentException("failed to convert " + propertyName + ". value: " + value);
+        logger.warn("No {} found for {}. prefix: {}",
```

Comment:
I wasn't able to find an explanation for this change (logging instead of throwing).
Do you mind adding a comment so that I know why this change was necessary?